src/app/routes/enrolment/services/revoke/revoke.service.ts

```diff
@@ -43,7 +43,7 @@ export class RevokeService {
   private handleRevokeResponse() {
     return (source) =>
       source.pipe(
-        map((value: boolean) => {
+        map((value: boolean | StatusList2021Credential) => {
```

Comment:
the revoke check for off-chain claims returns StatusList2021Credential, not boolean. No effect on anything with this change, but is more reflective of the truth.

Reply:
Maybe we should have a different handler for each response? What is best practice for this @dawidgil ?

Reply:
I think it's better to have different handler. Because when we came back to this in next few months we might not remembering that off-chain claims return StatusList.

Reply:
However if you want to leave as it is for now, then i'm ok with it.

Reply:
No, I think separate is better too! Added here.